cmd/stop.go

```diff
@@ -23,12 +23,16 @@ func newStopCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// stopCmd represents the stop command
 	var stopCmd = &cobra.Command{
 		Use:   "stop",
-		Short: "Stops the local Appsody docker container for your project",
-		Long: `Stop the local Appsody docker container for your project.
+		Short: "Stop the local running Appsody container.",
+		Long: `Stop the local running Appsody container for your project.
 
-Stops the docker container specified by the --name flag. 
-If --name is not specified, the container name is determined from the current working directory (see default below).
-To see a list of all your running docker containers, run the command "docker ps". The name is in the last column.`,
+By default, the command stops the Appsody container that was launched from the project in your current working directory. 
+To see a list of all your running Appsody containers, run the command "appsody ps". The names of your running Appsody projects are in the second column.`,
```

Comment:
Is it not obvious which column contains the names? Could be risky saying which column they're in, in case it changes (as it already has, judging by the original text!) If it's not obvious, does the column have a heading that you could use instead? Also, "containers" and "projects" are both used to mean the same thing in one sentence - suggest changing "projects" to "containers". eg:
"The container names are shown in the "blah" column."

Reply:
It is obvious which column contains the names, I have removed this sentence